packages/mc-scripts/src/utils/user-agent.ts

```diff
@@ -1,12 +1,12 @@
-const createHttpUserAgent = require('@commercetools/http-user-agent');
-const pkgJson = require('../../package.json');
+import createHttpUserAgent from '@commercetools/http-user-agent';
+import pkgJson from '../../package.json';
 
 const userAgent = createHttpUserAgent({
-  name: 'cli-login',
+  name: 'graphql-request',
```

Comment:
I think the previous name was more meaningful from the debugging perspective.

Reply:
The "cli-login" value is not correct because the user agent is for the entire CLI and not one single command.
The value here refers to the HTTP library we are using for sending requests, so in this case `graphql-request`. The user agent also contains the library name `mc-scripts`, so we know it's the CLI.